README.md

```diff
@@ -1,7 +1,7 @@
 # Music Source Separation in the Waveform Domain
 
 We provide an implementation of Demucs and Conv-Tasnet for music source separation on the [MusDB][musdb] dataset.
-They can separate drums, bass and vocals from the rest with state-of-the-art results, surpassing previous waveform or spectrogram based methods.
+They can separate drums, bass, and vocals from the rest with state-of-the-art results, surpassing previous waveform or spectrogram based methods.
```

Comment:
I think it was ok without the comma here

Reply:
I think it is a general rule of thumb to add comma before `and` when we are addressing multiple items.

Reply:
It's ok, too